src/blockUntilReady.test.ts

```diff
@@ -9,11 +9,11 @@ const metrics: Record<string | symbol, number> = {};
 
 const spy = new Proxy(redis, {
   get: (target, prop) => {
-    if (typeof metrics[prop] === "undefined") {
+    if (metrics[prop] === undefined) {
       metrics[prop] = 0;
     }
     metrics[prop]++;
-    // @ts-ignore - we don't care about the types here
+    // @ts-expect-error TODO: fix this
```

Comment:
is this something we want to fix in this PR? What's the issue?

Reply:
Not in this PR. The expect error message is kinda wrong. I will update that